pages/Açık Kaynak Felsefesi ve 6 Şubat Depremleri.md

```diff
@@ -0,0 +1,56 @@
+# Açık Kaynak Felsefesi ve 6 Şubat Depremleri
+Açık kaynak yazılımdan bahsettiğimizde genelde insanların zihinlerinde kaynak kodunun herkes tarafından görüntülenebileceği, değiştirilebileceği ve yeniden dağıtılabileceği yazılımlar sayesinde paylaşarak büyüyen bir ekosistem gelir. Yazılım geliştirme sürecinde saydamlığı, iş birliğini ve yenilikleri teşvik ederek daha güvenli, daha inovatif ve daha erişilebilir yazılımların oluşmasına katkı sağladığı için önem verdiğimiz açık kaynak felsefesinin, afet anında aynı amaç altında toplanan binlerce yazılımcının birlikte çalışmasını mümkün kılarak arama kurtarma organizasyonun sağlanmasında hatta insan hayatının kurtarılmasında kilit rol oynaması açık kaynağın çok daha fazlası olabileceğini düşünmemiz gerektiğini gösterdi.
+
+6 Şubat’ta gerçekleşen iki büyük deprem sonrasında saatler içerisinde Eser Özvataf ve Furkan Kılıç öncülüğünde her türlü geliştirici ve gönüllünün davet edildiği AYA: Açık Yazılım Ağı Discord kanalı kuruldu. Depremzedelerin yardımına koşmak için bir araya gelen birçok gönüllü geliştirici, açık kaynak yaklaşımıyla hareket ederek bu Discord kanalı üzerinden iletişim kurup ihtiyaç duyulan yazılımları hızlı bir şekilde işlevsel hale getirdi.
+
+Ve sadece 36 saat içerisinde afetbilgi.com, afetharita.com ve beniyiyim.com gibi 3 kocaman proje canlıya alındı. 
```

Comment:
Linkleri tıklanabilir yapmak için text'e link gömebilirsin.

Ve sadece 36 saat içerisinde [afetbilgi.com](https://afetbilgi.com), [afetharita.com](https://afetharita.com) ve [beniyiyim.com](https://beniyiyim.com) gibi 3 kocaman proje canlıya alındı.